metazoa/species/Anneissia/Anneissia_japonica/Anneissia_japonica_gca011630105v1_annotation.md

```diff
@@ -2,7 +2,7 @@
 ----------
 
 The annotation presented is derived from annotation submitted to
-[INSDC](http://www.insdc.org) with the assembly accession [GCA_011630105.1](http://www.ebi.ac.uk/ena/data/view/GCA_011630105.1).
+[INSDC](http://www.insdc.org) with the assembly accession [GCA\_011630105.1](http://www.ebi.ac.uk/ena/data/view/GCA_011630105.1).
```

Comment:
Im not entirely sure adding the backslash makes any difference to display or linking to URLs. It doesn't hurt to have them but I don't think they are necessitated for future ref

Reply:
It's better to escape special symbols. We had issues with unescaped _ in accession previously.